### PR TITLE
Closing date

### PR DESCRIPTION
**Pull request by gmackay**

### CODE REVIEW

File: vcm/src/test/java/uk/gov/cshr/vcm/repository/SearchQueryBuilderTest.java [hunk 1]
```diff
@@ -1,6 +1,10 @@
 package uk.gov.cshr.vcm.repository;
```
Comment (seamussingh) [inline vcm/src/test/java/uk/gov/cshr/vcm/repository/SearchQueryBuilderTest.java]: Have all the existing .sql snippets in test/resources been modified where required to include the extra AND clause for closing date?

File: vcm/src/main/java/uk/gov/cshr/vcm/controller/exception/VacancyException.java [hunk 2]
```diff
@@ -12,10 +12,6 @@
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper = false)
 public class VacancyException extends RuntimeException {
-
-//    private Long vacancyID;
-//    private HttpStatus status;
```
Comment (seamussingh) [inline vcm/src/main/java/uk/gov/cshr/vcm/controller/exception/VacancyException.java]: It is cleaner to delete not leave commented out code

Comment (reachsaurabh) [reply vcm/src/main/java/uk/gov/cshr/vcm/controller/exception/VacancyException.java]: Agreed , @gmackay integrate SonarLint and code coverage plugin into your ide to suggest , we will have Sonar and Jenkins one day to give the stats till that please do it locally as it becomes difficult to spot

Comment (gmackay) [reply vcm/src/main/java/uk/gov/cshr/vcm/controller/exception/VacancyException.java]: Left in by mistake.  You know you can edit files directly from GitHub/pull requests so please feel free to tidy up anything you notice :)

File: vcm/src/test/java/uk/gov/cshr/vcm/repository/SearchQueryBuilderTest.java (hunk 1 above)
Comment (gmackay) [reply vcm/src/test/java/uk/gov/cshr/vcm/repository/SearchQueryBuilderTest.java]: I have updated the unit test.

**Pull request merged**